### PR TITLE
feat(frontend): Update the design of the "Notifications" page tm-447

### PR DESCRIPTION
**Pull request by GEOFARL**
image

### CODE REVIEW

File: apps/frontend/src/pages/notifications/libs/components/notification-list-item/notification-list-item.tsx [hunk 1]
```diff
@@ -52,11 +57,21 @@ const NotificationListItem: React.FC<Properties> = ({
 					src={notification.userAvatarUrl ?? defaultAvatar}
 				/>
 			</Link>
-			<div>
+			<div className={styles["text-content"]}>
 				<div className={styles["notification-title"]}>
-					<span>{notification.message}</span>
+					{splitOnWord(notification.message, fullName).map(
```
Comment (what1s1ove) [inline apps/frontend/src/pages/notifications/libs/components/notification-list-item/notification-list-item.tsx:62]: let's avoid adding bold this time

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/notifications/styles.module.css [hunk 2]
```diff
@@ -1,49 +1,79 @@
...
+	width: 100%;
+	max-width: 637px;
+	height: calc(100% - 100px);
+}
+
+.divider {
+	flex-shrink: 0;
+	height: 1px;
+	margin: 30px 0;
+	background-color: rgb(174 180 187 / 15%);
```
Comment (fshabanov) [inline apps/frontend/src/pages/notifications/styles.module.css:36]: Can we use `rgba` here?

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

File: apps/frontend/src/pages/notifications/styles.module.css (hunk 2 above)
Comment (what1s1ove) [reply apps/frontend/src/pages/notifications/styles.module.css:36]: @fshabanov merge once it's done

**Review by fshabanov — APPROVED** [review]

File: apps/frontend/src/pages/notifications/styles.module.css [hunk 3]
```diff
@@ -1,51 +1,79 @@
...
+	width: 100%;
+	max-width: 637px;
+	height: calc(100% - 100px);
+}
+
+.divider {
+	flex-shrink: 0;
+	height: 1px;
+	margin: 30px 0;
+	background-color: rgba(174 180 187 / 15%);
```
Comment (fshabanov) [inline apps/frontend/src/pages/notifications/styles.module.css:36]:
```suggestion
	background-color: rgba(174, 180, 187, .15);
```

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/notifications/styles.module.css (hunk 3 above)
Comment (GEOFARL) [reply apps/frontend/src/pages/notifications/styles.module.css:36]: image

**Review by fshabanov — APPROVED** [review]

**Pull request merged**